### PR TITLE
build: fix rpi3 and hikey960 builds

### PR DESCRIPTION
**Pull request by igoropaniuk**
Based on build results https://travis-ci.org/OP-TEE/build/builds/271755688

`Signed-off-by: Igor Opaniuk <igor.opaniuk@linaro.org>`

### CODE REVIEW

File: hikey960.mk [hunk 1]
```diff
@@ -58,10 +58,12 @@ STRACE_PATH			?=$(ROOT)/strace
 # Targets
 ################################################################################
 .PHONY: all
-all: arm-tf boot-img lloader strace
+all: arm-tf boot-img lloader strace optee-examples
```
Comment (vchong) [inline hikey960.mk]: minor: not necessary to have the target here since `common.mk` already takes care of it in `filelist-tee-common`.

**Comment by vchong** [comment]
Other than the minor comment:

`Acked-by: Victor Chong <victor.chong@linaro.org>`

**Comment by jbech-linaro** [comment]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Comment by igoropaniuk** [comment]
Applied tags and rebased

File: hikey960.mk (hunk 1 above)
Comment (igoropaniuk) [reply hikey960.mk]: fixed

**Pull request merged**